cmodelres/CMakeLists.txt

```diff
@@ -129,3 +154,28 @@ if(SEA_EXE)
         COMMAND_EXPAND_LISTS
     )
 endif()
+
+set(CMAKE_C_COMPILER clang)
+set(LLVM_LINK llvm-link)
+set(KLEE_DRIVER_BC "klee_driver.bc")
+set(KLEE_SRC "/home/klee/klee_src/include")
+
+# target for KLEE
+if(KLEE_EXE)
+    add_custom_target(
+            klee_driver
+        COMMAND ${CMAKE_C_COMPILER} -I ${KLEE_SRC} -D MC_USE_STDINT -emit-llvm -c -g -O0 -Xclang -disable-O0-optnone ${CMAKE_CURRENT_SOURCE_DIR}/cmodel.c ${CMAKE_CURRENT_SOURCE_DIR}/libverify/verify_klee.c
+        COMMAND ${LLVM_LINK} -f cmodel.bc verify_klee.bc -o ${KLEE_DRIVER_BC}
+            COMMAND_EXPAND_LISTS
```

Comment:
@huxinwen1994 Maybe we need to add something like this here:

```
SOURCES ${KLEE_DEPS}
```

The list should be defined before adding the target. Something like this:

```
    set(KLEE_DEPS "")
    list(APPEND KLEE_DEPS "${CMAKE_CURRENT_SOURCE_DIR}/libverify/verify_klee.c")
    foreach(fn ${EXE_SRCS_C})
        list(APPEND KLEE_DEPS "${CMAKE_CURRENT_SOURCE_DIR}/${fn}")
    endforeach(fn)

    add_custom_target(
        klee_driver
    COMMAND ${CMAKE_C_COMPILER} -I ${KLEE_SRC} -D MC_USE_STDINT -emit-llvm -c -g -O0 -Xclang -disable-O0-optnone ${KLEE_DEPS}
        COMMAND ${LLVM_LINK} -f cmodel.bc verify_klee.bc -o ${KLEE_DRIVER_BC}
        COMMAND_EXPAND_LISTS
        SOURCES ${KLEE_DEPS}
    )
```